src/webnn_native/nnapi/NnapiUtils.h

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_NNAPI_UTILS_H_
+#define WEBNN_NATIVE_NNAPI_UTILS_H_
+#include <functional>
+#include <numeric>
+
+#include "webnn_native/Error.h"
+#include "webnn_native/Graph.h"
+#include "webnn_native/Operand.h"
+#include "webnn_native/Operator.h"
+
+#include "NeuralNetworksTypes.h"
+#include "nnapi_implementation.h"
+#include "webnn_native/nnapi/ErrorNnapi.h"
+
+namespace webnn_native { namespace nnapi {
+
+    struct NodeInfo {
+        int fd;
+        ANeuralNetworksMemory* mem;
+        ml::OperandType type;
+        std::vector<uint32_t> dimensions;
+        std::string name;
```

Comment:
Why do we need the name variable?

Reply:
Agree. Added for legacy reasons. Will remove